controllers/storagecluster/cephcluster.go

```diff
@@ -59,6 +59,12 @@ const (
 	clusterNetworkSelectorKey = "cluster"
 )
 
+const (
+	// PriorityClasses for cephCluster
+	systemNodeCritical    = "system-node-critical"
+	openshiftUserCritical = "openshift-user-critical"
```

Comment:
Have you checked to see if there's an import we could use that defines these for us?

Reply:
not really will check that out

Reply:
Did not found anything in our org 
https://github.com/search?q=org%3Aopenshift+openshift-user-critical&type=code
https://github.com/search?q=org%3Aopenshift+system-node-critical&type=Code

Reply:
Huh... great. :joy: All right!